x-pack/legacy/plugins/uptime/public/state/actions/monitor_duration.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { createAction } from 'redux-actions';
+import { QueryParams } from './types';
+
+export const getMonitorDurationAction = createAction<QueryParams>('GET_MONITOR_DURATION');
```

Comment:
I've noticed we have some actions named with `CAPS_SNAKE_CASE` and some like `CAPS WHITESPACE`.

It's not a high priority, but we should probably make them consistent. I prefer the snake-case way personally.